py-wacz/wacz/waczindexer.py

```diff
@@ -181,19 +181,20 @@ def serialize_cdxj_pages(self, pages):
 
     def serialize_json_pages(self, pages):
         yield json.dumps({'format': 'json-pages-1.0', 'title': 'All Pages'}) + "\n"
-
+        id = 0
         for line in pages.values():
```

Comment:
Let's use a unique id/uuid instead of incremental index when generating pages, to make it easier to combine pages from different sources.
Pages from Webrecorder/Conifer metadata should already have ids, I think, that could be used here.